core/domain/exp_services.py

```diff
@@ -308,6 +395,13 @@ def get_exploration_summaries_matching_ids(exp_ids):
     """Given a list of exploration ids, return a list with the corresponding
     summary domain objects (or None if the corresponding summary does not
     exist).
+
+    Args:
+        exp_ids: list of exploration ids.
+
+    Returns:
+        List of ExplorationSummary domain objects.
```

Comment:
A list of ExplorationSummary domain objects corresponding to the given exp_ids. If a given summary does not exist, None is returned in its place.